backend/models/group.js

```diff
@@ -221,6 +222,7 @@ groupSchema.methods.updateAttrs = function(data){
 			if (obj.shared_id) {
 				const ns = obj.account + "__" + obj.model;
 				if ("[object String]" === Object.prototype.toString.call(obj.id)) {
+					sharedIDSets.add(obj.shared_id);
```

Comment:
Should this line be in here? Shouldn't obj.shared_id always be added to sharedIDSets? This IF is just checking that obj.id is a String before attempting to convert it to a UUID.

Reply:
yup, good point